lib/Conversion/SeqToSV/SeqToSV.cpp

```diff
@@ -0,0 +1,86 @@
+//===- SeqToSV.cpp - Implement Seq passes -------------------------------===//
```

Comment:
I'm going to assume this is a simple file rename, though GH doesn't show it that way.